reactioncommerce/reaction
# Sent Email Closes Email Log Panel

Resolves [1985](https://github.com/reactioncommerce/reaction/issues/1985)

**Testing**
- Clone Branch and run `reaction reset` on Reaction's directory from your terminal
- Run Reaction
- Log In as an admin
- Add product to cart and `Checkout`
- Go to dashboard and select `Email`
- Enter you provider details in `Mail Provider` Panel and `save`
- Try resending the email in the `Email Logs` by clicking the `arrow` close to the red circle
- Observe that `Email Logs` Panel doesn't close after clicking the `arrow`
- Also Observe that when you resize browser, the `Email Logs` panel stays open
